Board meeting minutes for 2024-02-03 (#8)
cart/bevy-website: content/foundation/meeting-minutes/2024-02-03/index.md

**Cart for Treasurer.**
Proposed by: Cart

Unanimously approved, with discussion of this as a temporary position.

## Opening a bank account

Cart shall open a bank account on behalf of the Foundation.

Proposed by: Cart

Unanimously approved.

## Transferring of the Bevy logo copyright

The Bevy Foundation should accept a copyright transfer of the Bevy logo from Carter Anderson.

Proposed by: Cart

Unanimously approved.

## Making foundation board meeting minutes generally publicly available

The board should make these meeting notes publicly available at the time of publication.

Proposed by: Alice

Unanimously approved.

## Signing of meeting minutes

Discussed how to validate meeting minutes.

Agreed on storing meeting minutes on Github on `bevy-website repo`, and approved by board members via PR approval.

## Transferring the ownership of the bevyengine.org domain to the Foundation

Cart should transfer ownership of the bevyengine.org domain to the foundation.

Proposed by: James

Discussed how to pay for the bills associated. May need to delay until bank account is open and has funds.

> Cart proposed to transfer a modest sum (~$2000) to fund the account from his personal account. Will cover hosting costs with that money, and then will be reimbursed by an interest-free loan.

Author: Exact wording here needs to be clarified / corrected.

+++
title = "Board meeting: 2024-02-03"
date = 2024-02-03
[extra]
author = "Alice I. Cecile"
+++

<!-- more -->

## Attendees

- Rob Swain
- James Liu
- Carter Anderson

> - Alice Cecile

Author: ```suggestion
- Alice I. Cecile
```

- François Mockers

## Agenda

1. Adopt bylaws
2. Elect officers
3. Approve Cart opening a bank account
4. Approve Cart transferring copyright of the Bevy logo

## Adopt Bylaws

Proposed by: Cart

As defined in [Cart’s branch on 2024-02-03](https://github.com/cart/bevy-website/blob/01289396b15858715dd165956829fe00f92d50f0/content/foundation/bylaws/index.md).

Unanimously approved.

## Officers

**Cart for President.**

Proposed by: Cart

Unanimously approved.

**Alice for Secretary.**

Proposed by: Cart

Unanimously approved.

Board discussed who should be treasurer. Easier for US persons, due to the need to go in to the bank branch to validate ID.

> **Cart for Treasurer.**
> Proposed by: Cart

Reviewer: ```suggestion
**Cart for Treasurer.**

Proposed by: Cart
```

Unanimously approved, with discussion of this as a temporary position.

## Opening a bank account

Cart shall open a bank account on behalf of the Foundation.

Proposed by: Cart

Unanimously approved.

## Transferring of the Bevy logo copyright

The Bevy Foundation should accept a copyright transfer of the Bevy logo from Carter Anderson.

Proposed by: Cart

Unanimously approved.

## Making foundation board meeting minutes generally publicly available

The board should make these meeting notes publicly available at the time of publication.

Proposed by: Alice

Unanimously approved.

## Signing of meeting minutes

Discussed how to validate meeting minutes.

Agreed on storing meeting minutes on Github on `bevy-website repo`, and approved by board members via PR approval.

## Transferring the ownership of the bevyengine.org domain to the Foundation

Cart should transfer ownership of the bevyengine.org domain to the foundation.

Proposed by: James

Discussed how to pay for the bills associated. May need to delay until bank account is open and has funds.

> Cart proposed to transfer a modest sum (~$2000) to fund the account from his personal account. Will cover hosting costs with that money, and then will be reimbursed.

Author: ```suggestion
Cart will cover hosting costs personally initially, and then will be reimbursed.
```
